src/test/java/core/basesyntax/CsvFileServiceImplTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.CsvFileService;
+import core.basesyntax.dao.CsvFileServiceImpl;
+import java.io.File;
+import java.util.List;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CsvFileServiceImplTest {
+    private static final String OK_READ_FILE_PATH = "src/test/resources/CsvFileTestInput.csv";
+    private static final String OK_WRITE_FILE_PATH = "src/test/resources/FileWriteDaoTest.csv";
+    private static final String NOT_OK_FILE_PATH = "src/test/no such path";
+    private static final String STRING_TO_WRITE = "apple, banana";
+    private static final List<String> EXPECTED_LIST = List.of("apple", "banana");
```

Comment:
let's move this to the test method where it's used, expected result is better to declare in the methods

Reply:
Shall I move all 4 variables? I've relocated List EXPECTED_LIST since I'm using it only once. The rest of the Strings are used multiple times within the tests, shall I declare them in each relevant test?

Reply:
That was only concerning EXPECTED_LIST, it's ok to have generic variables in class that may be reused and are not test case specific